packages/c/calamares/files/install/modules/shellprocess_postinstall.conf

```
@@ -10,8 +10,8 @@ script:
       timeout: 300
 
     # Add a comment to the vconsole conf about how to change the font in the future
-    - "echo \"## The default console font on Solus is ter-v32b, if you would like to change that uncomment this and set it to your selection\" >> /etc/vconsole.conf"
-    - "echo \"# FONT=ter-v32b\" >> /etc/vconsole.conf"
+    - "echo \"## The default console font on Solus is ter-v32b, if you would like to change that, change it below:\" >> /etc/vconsole.conf"
+    - "echo \"FONT=ter-v32b\" >> /etc/vconsole.conf"
```

How about the following:

```suggestion
    - |
      cat << 'EOF' >> /etc/vconsole.conf
      ## The default console font on Solus is ter-v32b,
      ## it can be changed to a different font of your preference here.
      ## Run 'sudo clr-boot-manager update' after updating it.
      FONT=ter-v32b
      EOF
```